modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The {lcao} will do the following:
+
+* Checks for required system configurations
+* Performs any necessary system cleanup before generating the seed image
+* Launches the image generation tool, which does the following:
+** Stops the cluster operators
+** Prepares the seed image configuration
+** Generates and pushes the seed image to the image repository
+** Restores the cluster operators
+//** Prepares for precaching operations
+//** Backs up platform configurations and artifacts
+* Restores and updates the `SeedGenerator` CR, and restores the `ManagedCluster` CR on the hub cluster
+
```

Comment:
Probably add a note to warn users that the cluster will be up and running. But you are not suggested to upgrade using a regular way (oc adm upgrade) and create a new seed image from the new version and so on. cc/ @browsell @omertuc 

AFAIK the suggested way is to install and configure a new seed cluster from scratch. I would not include that sentence, but I will include that it is not suggested (supported?) to run multiple seed image creations of the same seed cluster,e .g: run seed image creation, upgrade seed cluster, run seed image creation, upgrade cluster again...